.github/workflows/build.yml

```diff
@@ -10,6 +10,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - name: Ensure the base branch is correct
+        if: github.ref != 'refs/heads/develop' && github.base_ref != 'develop'
```

Comment:
We need to ensure that builds that run on push to `master` (to perform a live deploy) skip this check.

Reply:
Okay, so what's the steps to test this do you think?